packages/core/src/scrim/Scrim.tsx

```diff
@@ -160,11 +153,11 @@ export const Scrim = forwardRef<HTMLDivElement, ScrimProps>(function Scrim(
         }
       }
     };
-  }, [containerFix, open, scrimId]);
+  }, [open, containerRef, scrimId]);
```

Comment:
Don't think ref value change would trigger effect?

Reply:
It won't, but eslint was complaining because it doesn't know it is a ref because it comes from props and not a local useRef.